cradle-cassandra/src/test/java/com/exactpro/cradle/cassandra/integration/BaseCradleCassandraTest.java

```diff
@@ -164,6 +169,22 @@ protected TestEventToStore generateTestEvent (String scope, Instant start, long
         return batch;
     }
 
+    @AfterMethod(description = "Verify that no tombstones was created")
+    public void ensureNoTombstones() {
```

Comment:
Could you please extract logic of this method into static method and reuse here or try to move this test into CassandraCradleHelper to execute it after suite. Some classes in future can don't extends BaseCradleCassandraTest

Reply:
Moved to custom listener class (TombstoneCounterListener)